### PR TITLE
new Vector3 functions

### PR DESCRIPTION
**Pull request by MarkCavalli**
https://github.com/altmp/v8-helpers/pull/4

### CODE REVIEW

File: server/index.d.ts [hunk 1]
```diff
@@ -417,7 +417,7 @@ declare module "alt-server" {
   export const DefaultDimension: number;
   export const GlobalDimension: number;
 
-  export class Vector3 {
+  export class Vector3 implements IVector3 {
```
Comment (Lhoerion) [inline server/index.d.ts:420]: Remove `implements IVector3` for now

Comment (MarkCavalli) [reply server/index.d.ts]: +

**Comment by Lhoerion** [comment]
Thanks for contribution in both repos, we'll deal later with adding proper comments as part of the bigger PR

**Review by Lhoerion — APPROVED** [review]
Thanks for contribution in both repos, we'll deal later with adding proper comments as part of the bigger PR

File: client/index.d.ts [hunk 2]
```diff
@@ -226,6 +226,39 @@ declare module "alt-client" {
...
+    public mul(vector: IVector3): Vector3;
+
+    public negative(): Vector3;
+    public normalize(): Vector3;
+    public distanceTo(vector: IVector3): Vector3;
+    public angleTo(vector: IVector3): Vector3;
+    public angleToDegrees(vector: IVector3): Vector3;
+    public toRadians(): Vector3;
+    public toDegrees(): Vector3;
+    public isInRange(vector: IVector3, range: number): Vector3;
```
Comment (FabianTerhorst) [inline client/index.d.ts:261]: this returns a bool not a vector3

**Pull request merged**